examples/server-worker/index.ts

```diff
@@ -53,23 +53,33 @@ app.post('/key-shares/generate', async (req: Request, res: Response) => {
 
   const ssvKeys = new SSVKeys(SSVKeys.VERSION.V3);
   const privateKey = await ssvKeys.getPrivateKeyFromKeystoreData(keystore, password)
-  const shares = await ssvKeys.buildShares(privateKey, operator_ids, operator_keys);
+  const encryptedShares = await ssvKeys.buildShares(privateKey, operator_ids, operator_keys);
+  // params to scan contract for the latest cluster snapshot to fill the payload data
+  const contractParams = {
+    ownerAddress: 'VALIDATOR_OWNER_ADDRESS',
+    contractAddress: 'SSV_CONTRACT_ADDRESS',
+    nodeUrl: 'ETH_NODE_URL',
+  };
+
+  // Build final web3 transaction payload and update keyshares file with payload data
   const payload = await ssvKeys.buildPayload(
-    ssvKeys.validatorPublicKey,
-    operator_ids,
-    shares,
-    123456789,
+    {
+      publicKey: ssvKeys.publicKey,
+      operatorIds,
+      encryptedShares,
+      123456789,
+    },
+    contractParams
```

Comment:
This example is not going to work properly.
First of all because scanning happens in expressjs route, so for each request it will happen again and again.
Also this example is used in a hamlet project when there's tens of requests per second to generate keyshares as part of testnet flow. This is why if we merge this, it will break testnet because it will then take decade to build all the keyshares for 120> validators.

We definitely should add support for keyshares splitting only mode, but let's postpone it until we discuss current state of project with Ariel.
Definitely should NOT merge this to v3 for now